app/assets/stylesheets/desktop/nilavu.css

```diff
@@ -273,11 +273,80 @@ body {
     background-image: url(c_icon-d.png);
   }
 
-  .settings_top {
-    border-bottom: 1px solid #e5e6ef;
-    margin-bottom: 20px;
+
+  .balance {
+  	text-align: center;
+  	height: 223px;
+  }
+  .balance span {
+  	font-size: 20px;
+  	margin-top: 0px;
+  	color: #a4a6af;
+  	text-align: center;
+  }
+  .balance h2 {
+  	font-size: 22px;
+  	color: #a4a6af;
+  }
+  .balance .green {
+  	color: #7be090;
+  	font-weight: bold;
+  }
+  .balance .blue, span.blue {
+  	color: #7cb4dd;
+  	font-weight: bold;
+  }
+
+
+  .usg {
+  	font-size: 22px;
+  	color: #a5a6af;
+  	padding-left: 0;
+  }
+  .center {
+  	display:flex;
+  	justify-content:center;
+  	align-items:center;
+  }
+
+  .table thead tr th {
```

Comment:
Change the name to as we discussed. either indented under `billing-history` or unique tag `billing-history-table`